src/components/LoanCards/Buttons/LoanFundedText.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+	<div class="loan-funded-text small-text">
+		This loan has already funded!<br>
+		Please choose another one.
+	</div>
+</template>
+
+<style lang="scss" scoped>
+@import 'settings';
+
+.loan-funded-text {
+	background-color: #f0f8f0;
```

Comment:
@foochris can you please add this to our _settings.scss file as a var so it will be reusable elsewhere.

Reply:
@mcstover  As a color var (e.x. 'light-green')?

Reply:
yes please

Reply:
PR updated with `_settings.scss` change.